src/openrct2.c

```diff
@@ -610,30 +610,6 @@ static bool openrct2_release_rct2_segment()
 {
 	bool result = true;
 #if defined(__unix__) || (defined(__APPLE__) && defined(__MACH__))
-	int len = 0x01429000 - 0x8a4000; // 0xB85000, 12079104 bytes or around 11.5MB
```

Comment:
Why is this being removed for linux? It still uses the mmap technique to setup the segments.